includes/admin/payments/class-order-adjustments-table.php

```diff
@@ -232,14 +218,37 @@ public function column_name( $order_adjustment ) {
 	 */
 	public function column_cb( $order_adjustment ) {
 		return sprintf(
-			'<input type="checkbox" name="%1$s[]" value="%2$s" />',
+			'<input type="checkbox" name="%1$s[]" id="%1$s-%2$s" value="%2$s" /><label for="%1$s-%2$s" class="screen-reader-text">%3%s</label>',
 			/*$1%s*/
 			'order_adjustment',
 			/*$2%s*/
-			$order_adjustment->id
+			$order_adjustment->id,
+			/* translators: discount type */
+			sprintf( __( 'Select %s', 'easy-digital-downloads' ), $this->get_order_adjustment_type_name( $order_adjustment ) )
 		);
 	}
 
+	/**
+	 * Gets a user friendly name/label for the order adjustment type
+	 *
+	 * @param \EDD\Orders\Order_Adjustment $order_adjustment Order Adjustment object.
+	 * @return string
+	 */
+	private function get_order_adjustment_type_name( $order_adjustment ) {
+		$name = '&mdash;';
```

Comment:
Because `$name` is only used at the end, you could remove this variable and just put this at the very end:

```php
return '&mdash;';
```